### PR TITLE
Sjip 88

### PR DESCRIPTION
**Pull request by oliviercperrier**
image

image

### CODE REVIEW

File: src/components/Layout/SideImage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  logoSrc?: string;
+  sideImgSrc?: string;
+  children: ReactElement;
+}
+
+const SideImageLayout = ({ logoSrc, sideImgSrc, children }: OwnProps) => (
+  <div className={style.sideImagePageContainer}>
+    {logoSrc && (
+      <a href={EnvVariables.configFor({ key: "INCLUDE_WEB_ROOT" })}>
+        <img className={style.logoImage} src={logoSrc} alt="Kids First Logo" />
```
Comment (celinepelletier) [inline src/components/Layout/SideImage/index.tsx:17]: Kids First => Include

File: src/ProtectedRoute.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  const { user } = { user: { roles: ["allo"], acceptedTerms: true } } as any; //useUser(); TODO
+  const { keycloak } = useKeycloak();
+  const userNeedsToLogin = !keycloak.authenticated;
+
+  if (userNeedsToLogin) {
+    return <Redirect to={STATIC_ROUTES.HOME} />;
+  }
+
+  if (!hasUserRole(user)) {
+    return <Redirect to={STATIC_ROUTES.JOIN} />;
```
Comment (celinepelletier) [inline src/ProtectedRoute.tsx:18]: Not sure the flow with Join page and Terms and Conditions will be served by the portal

File: src/provider/KeycloakProvider.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import EnvVariables from "helpers/EnvVariables";
+import { ReactKeycloakProvider as KeycloakProvider } from "@react-keycloak/web";
+import keycloak from "initKeycloak";
+
+export interface IProvider {
+  children: React.ReactNode;
+}
+
+const eventLogger = (eventType: AuthClientEvent, error?: AuthClientError) => {
+  if (EnvVariables.configFor({ key: "ENV" }) == "develpment" && error) {
```
Comment (celinepelletier) [inline src/provider/KeycloakProvider.tsx:15]: develpment => development

File: src/views/Home/index.tsx [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+  }
+
+  return (
+    <div className={styles.homePageContent}>
+      <div className={styles.loginContainer}>
+        <Space size={48} direction="vertical">
+          <div className={styles.logoTitleContainer}>
+            <IncludeIcon />
+            <Title level={4} className={styles.logotTitle}>
+              INCLUDE Data Hub
```
Comment (celinepelletier) [inline src/views/Home/index.tsx:57]: I guess we should use intl instead for static text

File: src/ProtectedRoute.tsx (hunk 2 above)
Comment (oliviercperrier) [reply src/ProtectedRoute.tsx:18]: I put it just in case, but we will remove it if its not necessary eventually

**Pull request merged**